### ISSUE TITLE
yellow and purple Xychorium/Xychoridite

### ISSUE THREAD

**Issue by ColonelGerdauf** [Soaryn/XyCraftTracker#19]
More colours... why not?

**Comment by Soaryn** [comment]
We could add literally all the colors under the sun. However, conceptually this would not be necessarily a grand idea. Instead we are giving a story to the 5 colors that exist. By default this is Red, Green, Blue, Dark, and Light. While I can't say for certain this will always be the case, I'd prefer to keep these to having a lore rather than just have a rainbow of everything.

That being said, in an upcoming version of XyCraft, you will be able to change those colors of the 5 pre-existing blocks with a resource pack simply. Note, this will not change the item texture colors and such, but over all it will allow external modifications.

TL;DR: I have a story of these materials thus far, and unless it was condensed into 1 item instead of 5, the colors will remain as they are.
